src/services/uploader/dto/start-encode.dto.ts

```diff
@@ -5,5 +5,17 @@ export class StartEncodeDto {
     description: 'ID of the upload',
     default: 'ec102517-7be9-4255-9d07-75a525a88565',
   })
-  upload_id: string
+  upload_id: string;
+
+  @ApiProperty({
+    description: 'Video ID',
+    default: 'ec102517-7be9-4255-9d07-75a525a88565',
+  })
+  video_id: string;
+
+  @ApiProperty({
+    description: 'permlink of the post',
+    default: 'ec102517-7be9-4255-9d07-75a525a88565',
```

Comment:
Could this be generated using uuid? There's a risk of multiple videos getting the same id or permlink